Competition2017/src/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -148,7 +163,6 @@ public void tankDrivePowerMode(double leftPower, double rightPower) {
      * @param motor used to access the motor to set and give data
      */
     private void updateMotorConfig(XCANTalon motor) {
-        motor.configEncoderCodesPerRev((int)encoderCodesProperty.get());
```

Comment:
Idk he just told me to remove this and I think this has nothing to do with the inch conversion function.